src/main/java/com/snowflake/kafka/connector/Utils.java

```diff
@@ -329,7 +335,7 @@ static boolean isValidSnowflakeTableName(String tableName) {
    * @return connector name
    */
   static String validateConfig(Map<String, String> config) {
-    boolean configIsValid = true; // verify all config
+    Map<String, String> invalidConfigParams = new HashMap<String, String>(); // verify all config
```

Comment:
this is the crux of the change, replacing the configIsValid boolean with a map of invalid params